spec/features/recipes/show_recipe_page_spec.rb

```diff
@@ -0,0 +1,22 @@
+feature 'user visits show recipe page' do
+  before do
+    FactoryBot.create(:recipe, :scrambled_eggs)
+    FactoryBot.create(:recipe, :tikka_masala)
+    visit('/')
+  end
+
+  it 'shows user clicked recipe' do
+    page.all(:xpath, '//a[contains(text(), "Show")]')[0].click
```

Comment:
1. Should be moved to `before` block and put right after our `visit` call.
2. Can we please click second element `[1]`? and update the assertions below to check for `tikka_masala` stuff?